insights/combiners/mounts.py

```diff
@@ -42,6 +43,7 @@ class Mounts(object):
     """
 
     def __init__(self, binmount, procmounts, mountinfo):
+        deprecated(Mounts, "Use the parsers in insights.mount module instead", "3.1.0")
```

Comment:
@chenlizhong Please make it `3.1.25` which means we can remove it after the release of `3.2.0` which is roughly six months from now.

Reply:
Updated